CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 - Add support to select and load multiple files at once ([#257](https://github.com/cbrnr/mnelab/pull/257) by [Florian Hofer](https://github.com/hofaflo))
 - Add drag and drop reordering to sidebar ([#261](https://github.com/cbrnr/mnelab/pull/261) by [Florian Hofer](https://github.com/hofaflo))
 - Add support for plotting evoked potentials averaged over channels (Plot -> Evoked comparison...) ([#256](https://github.com/cbrnr/mnelab/pull/256) by [Florian Hofer](https://github.com/hofaflo))
+- Add "Details..." button to "Select XDF Stream" dialog ([#266](https://github.com/cbrnr/mnelab/pull/266) by [Florian Hofer](https://github.com/hofaflo))
+
 
 ### Changed
 - Simplify rereferencing workflow ([#258](https://github.com/cbrnr/mnelab/pull/258) by [Florian Hofer](https://github.com/hofaflo))
+- Move "Show information..." from toolbar to "File" menu and rename to "Show XDF meta information" ([#266](https://github.com/cbrnr/mnelab/pull/266) by [Florian Hofer](https://github.com/hofaflo))
```

Comment:
```suggestion
- Move "Show information..." from toolbar to "File" menu and rename to "Show XDF meta information..." ([#266](https://github.com/cbrnr/mnelab/pull/266) by [Florian Hofer](https://github.com/hofaflo))
```

Reply:
Oh right, that was a conscious decision, but I forgot to mention it: According to [this](https://docs.microsoft.com/en-us/windows/win32/uxguide/cmd-menus#using-ellipses) the ellipsis indicates further options will be presented once the action is selected, which isn't the case here (the "Details..." label is wrong in that case).

Reply:
Right, I always used it when an additional dialog pops up. Could you go over all menu entries to make that consistent (maybe in another PR)? We should leave the ellipsis for Preferences... though - that's standard on macOS but the Windows docs advise against it.

I think I added it because "Show XDF chunks..." requires additional inputs and therefore the ellipsis, which is not the case for "Show XDF meta information".

Reply:
So I guess in this PR the only thing left is change "Details..." to "Details" then?

Reply:
Yes, done!